Sources/LanguageServerProtocol/Requests/ShutdownRequest.swift

```diff
@@ -16,9 +16,12 @@
 /// not reach the client). See `Exit`.
 ///
 /// - Returns: Void.
-public struct Shutdown: RequestType, Hashable {
+public struct ShutdownRequest: RequestType, Hashable {
   public static let method: String = "shutdown"
   public typealias Response = VoidResponse
 
   public init() { }
 }
+
+@available(*, deprecated, message: "Use ShutdownRequest instead")
```

Comment:
I think you can do `@available(*, deprecated, renamed: "ShutdownRequest")` and it will provide a fixit :-)

Reply:
Ah, nice improvement. Thanks!